src/Item_Ticket.php

```diff
@@ -368,894 +154,20 @@ public static function showItemToAdd($tickets_id, $itemtype, $items_id, $options
      *
      * @return void
      **/
-    public static function showForTicket(Ticket $ticket)
+    public static function showForObject($ticket, $options = [])
```

Comment:
Proposed PR will probably be integrated in GLPI 10.1, which is an intermediate version (it is too late for GLPI 10.0).
In intermediate version, we try to not introduce BC breaks., so, if you want to rename a method, you have to handle a deprecation state.

For instance:
```suggestion
    public static function showForTicket(Ticket $ticket)
    {
        Toolbox::deprecated();
        static::showForObject($ticket);
    }

    public static function showForObject($ticket, $options = [])
```